cmd/policy-attributes.go

```diff
@@ -89,7 +97,7 @@ func policy_listAttributes(cmd *cobra.Command, args []string) {
 	}
 
 	t := cli.NewTable()
-	t.Headers("Id", "Namespace", "Name", "Rule", "Values", "Active")
+	t.Headers("Id", "Namespace", "Name", "Rule", "Values", "Active", "Metadata.Labels", "Metadata.CreatedAt", "Metadata.UpdatedAt")
```

Comment:
Nit: we could probably drop the metadata prefix and just call the columns "labels", "createdAt" and "updatedAt"